consensus/polybft/fsm.go

```diff
@@ -33,6 +33,8 @@ var (
 	errCommitEpochTxSingleExpected = errors.New("only one commit epoch transaction is allowed in an epoch ending block")
 	errProposalDontMatch           = errors.New("failed to insert proposal, because the validated proposal " +
 		"is either nil or it does not match the received one")
+	pbftDifficulty = uint64(0)
+	pbftNonce      = types.Nonce{}
```

Comment:
Not sure if we need this variable. Maybe just compare against `types.Nonce{}`
```suggestion
```

Reply:
I think we need this variable, i'm not sure that is possible to direct compare `types.Nonce{}`